cfgov/preprocessed/css/organisms/full-width-well.less

```diff
@@ -0,0 +1,45 @@
+/* topdoc
+  name: Bullet Points for full width text
+  family: cfgov-organisms
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="o-full-width-text">
+            <ul>
+              <li>Gray Well</li>
+              <li>Well's content</li>
+            </ul>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .o-full-width-text
+            ul
+              li
+  tags:
+    - cfgov-organisms
+  notes:
+    - Since content will be entered by content managers,
+      these styles are applied on the ul li elements directly when needed.
+*/
+
+.o-full-width-text {
+    ul {
+        padding-left: 30px;
+
+        li {
+          @font-size: 16px;
```

Comment:
Pointing out, the reason it's used elsewhere is because the `@font-size` is exposed by a mixin and the value changes based on breakpoints. That doesn't seem to be a need here though, so it's not necessary.